src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");

```

Comment:
The message in the `doWork` method should indicate what type of `Machine` is working. Consider specifying that it is a Bulldozer in a more informative way, as per the checklist item on writing informative messages in methods.